circle.yml

```diff
@@ -0,0 +1,26 @@
+machine:
+  services:
+    - docker
+
+dependencies:
+  pre:
+      # set build status to pending on start
+    - ./util/docker/maketest.sh --pending ${GH_TOKEN}
+      # Login to docker
+    - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
+    - ./util/docker/baseimage/buildbaseimage.sh
+      # Actually the test step, but we want to cache it so it can go here.
+    - ./util/docker/circle/buildcircle.sh
+  cache_directories:
+    - ~/.ccache
+
+test:
+  override:
+    - sh -c true
+
+deployment:
+  github:
+    branch: master
```

Comment:
If you want to test out the documentation updating, you can change this to your branch for now to have it run and see that it works.  It's fine if our docs are non-master for a few days :)